lighthouse-core/audits/audit.js

```diff
@@ -97,3 +151,14 @@ class Audit {
 }
 
 module.exports = Audit;
+
+/** @typedef {{
+ *     key: string,
+ *     itemType: string,
+ *     text: string,
+ * }}
+ */
+Audit.Heading; // eslint-disable-line no-unused-expressions
+
+/** @typedef {!Array<!Audit.Heading>} */
```

Comment:
Actually yeah. Basically we pass around this typedef and never use the Audit.Heading one directly. Not entirely worth it to nuke this guy, i tried.